src/Chainweb/CutDB.hs

```diff
@@ -266,26 +284,28 @@ startCutDb
 startCutDb config logfun headerStore payloadStore = mask_ $ do
     cutVar <- newTVarIO (_cutDbConfigInitialCut config)
     queue <- newEmptyPQueue
-    cutAsync <- asyncWithUnmask $ \u -> u $ processor queue cutVar
+    peerHeights <- newTVarIO . PeerHeights $ BQ.singleton 16 (BlockHeight 0)
```

Comment:
Unsure how big the Queue should be.

Reply:
Why not use `TBQueue` in this case?

Reply:
There's no way to fold over a `TBQueue`.